src/main/resources/db/migration/courtcase/V2129__add_personid_to_defendant.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+    alter table DEFENDANT add column PERSON_ID UUID NULL;
+
+    update DEFENDANT SET PERSON_ID = uuid_generate_v4() where PERSON_ID is null;
+
```

Comment:
Could we follow up with an `ALTER` to add in a `NOT NULL` constraint once the `NULLS` have been populated?